operator/config/prometheus/monitor.yaml

```diff
@@ -1,11 +1,11 @@
-
 # Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-    app: rhacs-operator
```

Comment:
Why removed, isn't it good practice to (logically) bundle resources by adding `app:` labels?

Reply:
Right, removing this was probably a mistake. Note we don't ship this resource anyway.